helm/oncall/values.yaml

```diff
@@ -3,7 +3,7 @@
 # Set the domain name Grafana OnCall will be installed on.
 # If you want to install grafana as a part of this release make sure to configure grafana.grafana.ini.server.domain too
 base_url: example.com
-base_url_protocol: https
+base_url_protocol: http
```

Comment:
`base_url_protocol` isn't the change from `https` to `http` a breaking change and maybe a security downgrade?